srcpkgs/dragon-player/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'dragon-player'
 pkgname=dragon-player
 version=19.04.2
-revision=1
+revision=2
 wrksrc="dragon-${version}"
 build_style=cmake
 hostmakedepends="extra-cmake-modules pkg-config qt5-qmake qt5-host-tools
- kcoreaddons kdoctools"
+ kcoreaddons kdoctools ksolid-devel"
```

Comment:
Why is `ksolid-devel` needed in `hostmakedepends` here?

Reply:
good catch, that was unintended